### ISSUE TITLE
Sources for disambiguation of variants

### ISSUE THREAD

**Issue by LinguList** [lexibank/chacolanguages#10]
@Bridnicolas, I only now realized that you distinguished explicit sources in some cases in the EDICTOR, right? We need to account for these on the word form level. Can you provide one example for a concept and a language, so I can check this in the EDICTOR? Ideally, we can standardize the display of variants then!

**Comment by LinguList** [comment]
I consider this some kind of a bonus, but we could also just submit with what I have written there now in the language part and later tackle this, if you prefer.

**Comment by Bridnicolas** [comment]
Of course. Do you mean cases like Kadiwéu 'ant', which is 'dibixoGo' as it appears in the official Griffiths source, but 'dibico' in Sándalo's grammar, which is what I mentioned under the Comments column? That could be just a variant, though. 
Another example: Chorote 'altar' is "t'ajsaa', but it only appears on IDS (provided by Gerzenstein), not on Drayson's dictionary.
Wichí also has many different sources. 'ashes' for instance is 'itoj mukw' on DIWICA (the online Wichí dictionary), but Nercesian's grammar also provides "muq'utaj" (one is literally 'fire dust' and the other is 'big (or false) dust')
Is that what you meant?

**Comment by LinguList** [comment]
Ashes is a good example, it shows, that the information on individual words was given in prose rather than in a text that I could interpret automatically (note contains not the source key, but rather a mixture of abbreviations, plus additional info). This means, that the information on the concrete source must be given in another run where the data would have to be checked when updating the database. 

I did not pay attention to this, but generally, my recommendation is not to mix sources, before one is sure about phonology etc., and when doing so to annotate this very clearly, e.g., by using the exact bibtex key with the page number in the NOTE field or an extra field for the source.

We leave this issue open until fixed, but leave it unchanged for the submission of the study.

**Comment by Bridnicolas** [comment]
Okay. That means, that we may submit the article and then fix it for future readers? 
I agree with your recommendation. Fortunately, thanks to the albeit unclear information under Notes, we may crack it down for clear annotation, but it will take some days and me being back from the field trip. 
When do you think we should start correcting this?

**Comment by LinguList** [comment]
Maybe, if reviews require it.

**Comment by LinguList** [comment]
So first, we leave this open and simply wait.

**Comment by FredericBlum** [comment]
@Bridnicolas @LinguList Is this something still being worked on?

**Comment by LinguList** [comment]
It is not resolved, should stay open.

**Comment by Bridnicolas** [comment]
Hi there! Yes, I agree. It would be nice to continue working on it some time.
